CefSharp/IBrowserHost.cs

```diff
@@ -11,6 +11,14 @@ public interface IBrowserHost : IDisposable
     {
         void StartDownload(string url);
         void Print();
+        /// <summary>
+        /// Prints the current browser contents to the PDF file and executes a callback
+        /// on completion. The caller is responsible for deleting the file when done.
+        /// </summary>
+        /// <param name="path">PDF file location.</param>
+        /// <param name="settings">Settings.</param>
+        /// <param name="callback">The callback executed on completion.</param>
+        void PrintToPDF(string path, CefSharpPdfPrintSettings settings, IPrintToPdfCallback callback);
```

Comment:
Exceptions are expensive and in this case unnecessary, I'd rather return a bool. Fair point about the path.